--- runtime/tray/akonaditray.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/tray/akonaditray.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -14,6 +14,7 @@
 Name[en_GB]=Akonaditray
 Name[es]=Akonaditray
 Name[et]=Akonaditray
+Name[fi]=Akonaditray
 Name[fr]=Miniature Akonadi
 Name[ga]=Akonaditray
 Name[gl]=Akonaditray
@@ -51,6 +52,7 @@
 GenericName[en_GB]=Akonadi Tray Utility
 GenericName[es]=Utilidad de bandeja de sistema de Akonadi
 GenericName[et]=Akonadi süsteemse salve tööriist
+GenericName[fi]=Akonadin ilmoitusaluetyökalu
 GenericName[fr]=Utilitaire miniature Akonadi
 GenericName[ga]=Uirlis Tráidire Akonadi
 GenericName[gl]=Utilidade da bandexa de Akonadi
--- runtime/plugins/akonadi_serializer_microblog.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/plugins/akonadi_serializer_microblog.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[en_GB]=Microblog Serialiser
 Name[es]=Serializador de Microblog
 Name[et]=Mikroblogi jadasti
+Name[fi]=Microblog-sarjoittaja
 Name[fr]=Sérialiseur Microblog
 Name[ga]=Srathóir Micreabhlagadóireachta
 Name[gl]=Serializador de microblogues
@@ -43,6 +44,7 @@
 Comment[en_GB]=An Akonadi serialiser plugin for Microblog
 Comment[es]=Un complemento serializador de Akonadi para Microblog
 Comment[et]=Akonadi mikroblogi jadastamisplugin
+Comment[fi]=Akonadi-sarjoittajaliitännäinen Microblogiin
 Comment[fr]=Un module externe Akonadi pour la sérialisation Microblog
 Comment[ga]=Breiseán srathóra Akonadi le haghaidh micreabhlagadóireachta
 Comment[gl]=Un engadido de serialización de Akonadi para Microblog
--- runtime/plugins/akonadi_serializer_mail.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/plugins/akonadi_serializer_mail.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[en_GB]=Mail Serialiser
 Name[es]=Serializador de correo
 Name[et]=Kirjade jadasti
+Name[fi]=Sähköpostisarjoittaja
 Name[fr]=Sérialiseur de courriels
 Name[ga]=Srathóir Ríomhphoist
 Name[gl]=Serializador de correo
@@ -45,6 +46,7 @@
 Comment[en_GB]=An Akonadi serialiser plugin for mail objects
 Comment[es]=Un complemento serializador de Akonadi para objetos correo
 Comment[et]=Akonadi kirjaobjektide jadastamisplugin
+Comment[fi]=Akonadi-sarjoittajaliitännäinen sähköpostiobjekteille
 Comment[fr]=Un module externe Akonadi pour la sérialisation des courriels
 Comment[ga]=Breiseán srathóra Akonadi le haghaidh ríomhphoist
 Comment[gl]=Un engadido de serialización de Akonadi para obxectos correo
--- runtime/migration/kres/kresmigrator.h	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/migration/kres/kresmigrator.h	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -114,7 +114,7 @@
       KConfigGroup resMigrationCfg( KGlobal::config(), "Resource " + resId );
       const QString akoResId = resMigrationCfg.readEntry( "ResourceIdentifier", "" );
       if ( akoResId.isEmpty() ) {
-        emit message( Error, "No Akonadi agent identifier specified for previously bridged resource '" + resId + "'" );
+        emit message( Error, i18n("No Akonadi agent identifier specified for previously bridged resource '%1'", resId ));
         migrateNext();
         return;
       }
@@ -126,7 +126,7 @@
       mBridgeManager = new KRES::Manager<T>( mType );
       mBridgeManager->readConfig( mConfig );
       if ( !mBridgeManager->standardResource() ) {
-        emit message( Error, "Bridged resource '" + resId + "' has no standard resource." );
+        emit message( Error, i18n("Bridged resource '%1' has no standard resource.", resId) );
         migrateNext();
         return;
       }
--- runtime/resources/nntp/nntpresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/nntp/nntpresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=Usenet Newsgroups (NNTP)
 Name[es]=Grupos de noticias Usenet (NNTP)
 Name[et]=Useneti uudistegrupid (NNTP)
+Name[fi]=Usenet-keskusteluryhmät (NNTP)
 Name[fr]=Forum de discussion usenet (NNTP)
 Name[gl]=Grupos de noticias da Usenet (NNTP)
 Name[hu]=Usenet-hírcsoportok (NNTP)
@@ -46,6 +47,7 @@
 Comment[en_GB]=Makes it possible to read articles from a news server
 Comment[es]=Hace posible leer artículos desde un servidor de noticias
 Comment[et]=Võimaldab lugeda artikleid uudisteserverist
+Comment[fi]=Mahdollistaa keskusteluryhmäpalvelimen artikkelin lukemisen
 Comment[fr]=Permet de lire des articles depuis un serveur de nouvelles
 Comment[gl]=Permite ler artigos desde un servidor de novas
 Comment[hu]=Híreket tud letölteni NNTP-hírkiszolgálókról
--- runtime/resources/pop3/pop3resource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/pop3/pop3resource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -8,6 +8,7 @@
 Name[en_GB]=POP3
 Name[es]=POP3
 Name[et]=POP3
+Name[fi]=POP3
 Name[fr]=POP3
 Name[gl]=POP3
 Name[it]=POP3
@@ -38,6 +39,7 @@
 Comment[en_GB]=Fetches mail from a POP3 server
 Comment[es]=Recupera correo desde un servidor POP3
 Comment[et]=Kirjade tõmbamine POP3 serverist
+Comment[fi]=Noutaa postin POP3-palvelimela
 Comment[fr]=Récupère les courriels depuis un serveur POP3
 Comment[gl]=Obtén correo dun servidor de POP3
 Comment[it]=Scarica messaggi da un server POP3
--- runtime/resources/pop3/accountdialog.cpp	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/pop3/accountdialog.cpp	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -245,7 +245,6 @@
     connect ( requestJob, SIGNAL(result(KJob*)),
               this, SLOT(localFolderRequestJobFinished(KJob*)) );
   }
-  folderRequester->setEnabled( false );
 
   if ( Settings::storePassword() ) {
     mWallet = Wallet::openWallet( Wallet::NetworkWallet(), winId(),
@@ -673,7 +672,6 @@
 void AccountDialog::targetCollectionReceived( Akonadi::Collection::List collections )
 {
   folderRequester->setCollection( collections.first() );
-  folderRequester->setEnabled( true );
 }
 
 void AccountDialog::localFolderRequestJobFinished( KJob *job )
@@ -683,7 +681,6 @@
     Q_ASSERT( targetCollection.isValid() );
     folderRequester->setCollection( targetCollection );
   }
-  folderRequester->setEnabled( true );
 }
 
 #include "accountdialog.moc"
--- runtime/resources/knut/knutresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/knut/knutresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -11,6 +11,7 @@
 Name[eo]=Knut
 Name[es]=Knut
 Name[et]=Knut
+Name[fi]=Knut
 Name[fr]=Knut
 Name[ga]=Knut
 Name[gl]=Knut
--- runtime/resources/maildir/maildirresource.cpp	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/maildir/maildirresource.cpp	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -346,7 +346,7 @@
 {
   kDebug() << collection << source << dest;
   if ( collection.parentCollection() == Collection::root() ) {
-    emit error( i18n( "Cannot move root maildir folder '%1'." ).arg( collection.remoteId() ) );
+    emit error( i18n( "Cannot move root maildir folder '%1'." ,collection.remoteId() ) );
     changeProcessed();
     return;
   }
--- runtime/resources/imap/imapaccount.cpp	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/imap/imapaccount.cpp	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -294,7 +294,6 @@
     emit error( capJob->session(),
                 CapabilitiesTestError,
                 i18n( "Could not test the capabilities supported by the IMAP server %1.", m_server ) );
-    disconnect();
     return;
   }
 
@@ -323,7 +322,6 @@
                 IncompatibleServerError,
                 i18n( "Cannot use the IMAP server %1, some mandatory capabilities are missing: %2. "
                       "Please ask your sysadmin to upgrade the server.", m_server, missing.join( ", " ) ) );
-    disconnect();
     return;
   }
 
--- runtime/resources/imap/imapresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/imap/imapresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -8,6 +8,7 @@
 Name[en_GB]=IMAP E-Mail Server
 Name[es]=Servidor de correo IMAP
 Name[et]=IMAP e-posti server
+Name[fi]=IMAP-sähköpostipalvelin
 Name[fr]=Serveur de courriels IMAP
 Name[gl]=Servidor de correo IMAP
 Name[it]=Server di posta IMAP
@@ -39,6 +40,7 @@
 Comment[en_GB]=Connects to an IMAP e-mail server.
 Comment[es]=Conecta a un servidor de correo IMAP.
 Comment[et]=Ühendumine IMAP e-posti serveriga.
+Comment[fi]=Yhdistää IMAP-sähköpostipalvelimeen.
 Comment[fr]=Se connecte à un serveur de courriels IMAP
 Comment[gl]=Conéctase a un servidor de correo IMAP.
 Comment[it]=Si collega ad un server di posta IMAP.
--- runtime/resources/birthdays/birthdaysresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/birthdays/birthdaysresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[en_GB]=Birthdays & Anniversaries
 Name[es]=Cumpleaños y aniversarios
 Name[et]=Sünni- ja aastapäevad
+Name[fi]=Syntymä- ja vuosipäivät
 Name[fr]=Anniversaires et fêtes
 Name[gl]=Cumpreanos e aniversarios
 Name[hu]=Születésnapok és évfordulók
@@ -43,6 +44,7 @@
 Comment[en_GB]=Provides access to birthday and anniversary dates of contacts in your address book as calendar events
 Comment[es]=Provee acceso a las fechas de cumpleaños  y aniversarios de los contactos en la libreta de direcciones como eventos de calendario
 Comment[et]=Võimaldab kasutada KDE aadressiraamatusse kalendrisündmustena salvestatud kontaktide sünni- ja aastapäevi
+Comment[fi]=Noutaa yhteystietojesi syntymä- ja vuosipäivät osoitekirjastasi kalenteritapahtumina
 Comment[fr]=Fourni l'accès aux dates d'anniversaire et de fête des contacts du carnet d'adresses KDE comme des évènements du calendrier
 Comment[gl]=Permite acceder ás datas de cumpreanos e aniversarios dos contactos do caderno de enderezos como eventos do calendario
 Comment[hu]=Hozzáférést biztosít a névjegyekben tárolt születésnapokhoz és évfordulókhoz a címjegyzékekben (naptári eseményként)
--- runtime/resources/vcarddir/vcarddirresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/vcarddir/vcarddirresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=VCard Directory
 Name[es]=Directorio VCard
 Name[et]=VCardide kataloog
+Name[fi]=vCard-kansio
 Name[fr]=Dossier VCard
 Name[ga]=Comhadlann v-Chárta
 Name[gl]=Directorio de VCard
@@ -48,6 +49,7 @@
 Comment[en_GB]=Loads data from a directory with VCards
 Comment[es]=Carga datos de un directorio con archivos VCard
 Comment[et]=Andmete laadimine VCardide kataloogist
+Comment[fi]=Noutaa tietoa kansiosta vCardeilla
 Comment[fr]=Charge des données depuis un dossier contenant des fichiers VCard
 Comment[ga]=Gníomhaire a luchtaíonn sonraí ó chomhadlann ina bhfuil v-Chártaí
 Comment[gl]=Carregar datos desde un cartafol con VCards 
--- runtime/resources/vcard/vcardresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/vcard/vcardresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=VCard File
 Name[es]=Archivo VCard
 Name[et]=VCard-fail
+Name[fi]=vCard-tiedosto
 Name[fr]=Fichier VCard
 Name[ga]=Comhad v-Chárta
 Name[gl]=Ficheiro de VCard
@@ -48,6 +49,7 @@
 Comment[en_GB]=Loads data from a VCard file
 Comment[es]=Carga datos de un archivo VCard
 Comment[et]=Andmete laadimine VCard-failist
+Comment[fi]=Noutaa tietoa vCard-tiedostosta
 Comment[fr]=Charge des données depuis un fichier au format VCard
 Comment[ga]=Breiseán a luchtaíonn sonraí ó chomhad v-Chárta
 Comment[gl]=Carrega datos desde un ficheiro VCard
--- runtime/resources/kabc/kabcresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/kabc/kabcresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[en_GB]=KDE Address Book (traditional)
 Name[es]=Libreta de direcciones de KDE (tradicional)
 Name[et]=KDE aadressiraamat (traditsiooniline)
+Name[fi]=KDE:n osoitekirja (perinteinen)
 Name[fr]=Carnet d'adresse KDE (traditionnel)
 Name[ga]=Leabhar Seoltaí KDE (traidisiúnta)
 Name[gl]=Caderno de enderezos do KDE (tradicional)
@@ -44,6 +45,7 @@
 Comment[en_GB]=Loads data from a traditional KDE address book resource
 Comment[es]=Carga datos desde un recurso tradicional de libreta de direcciones de KDE
 Comment[et]=Andmete laadimine traditsioonilisest KDE aadressiraamatu ressursist
+Comment[fi]=Noutaa tietoa KDE:n perinteisestä osoitekirjaresurssista
 Comment[fr]=Charge des données depuis un carnet d'adresses KDE traditionnel
 Comment[ga]=Breiseán a luchtaíonn sonraí ó acmhainn thraidisiúnta leabhair seoltaí KDE
 Comment[gl]=Carrega datos desde un recurso de caderno de enderezos do KDE
--- runtime/resources/kabc/kresourceassistant.cpp	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/kabc/kresourceassistant.cpp	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -370,7 +370,7 @@
     d->mConfigWidget->loadSettings( d->mCreationWidget->mResource );
   } else if ( item->widget() == d->mConfigWidget ) {
     d->mConfigWidget->saveSettings( d->mCreationWidget->mResource );
-
+    d->setReadOnly( d->mCreationWidget->mResource->readOnly() );
     const QString resourceName = d->mCreationWidget->mResource->resourceName();
     d->mFolderWidget->mName->setText( resourceName );
   }
--- runtime/resources/mbox/mboxresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/mbox/mboxresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[eo]=Mbox
 Name[es]=Mbox
 Name[et]=Mbox
+Name[fi]=Mbox
 Name[fr]=Mbox
 Name[ga]=Mbox
 Name[gl]=Mbox
@@ -42,6 +43,7 @@
 Comment[en_GB]=Loads data from a local mbox file
 Comment[es]=Carga datos de un archivo mbox local
 Comment[et]=Andmete laadimine kohalikust mbox-failist
+Comment[fi]=Noutaa tietoa paikallisesta mbox-tiedostosta
 Comment[fr]=Charge des données depuis un fichier mbox
 Comment[ga]=Breiseán a luchtaíonn sonraí ó chomhad logánta mbox
 Comment[gl]=Carrega datos desde un ficheiro mbox local
--- runtime/resources/openchange/ocresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/openchange/ocresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=OpenChange
 Name[es]=OpenChange
 Name[et]=OpenChange
+Name[fi]=OpenChange
 Name[fr]=OpenChange
 Name[ga]=OpenChange
 Name[gl]=OpenChange
@@ -47,6 +48,7 @@
 Comment[en_GB]=Makes it possible to work with an OpenChange / Exchange server
 Comment[es]=Hace posible trabajar con un servidor OpenChange / Exchange
 Comment[et]=Võimaldab kasutada OpenChange'i/Exchange'i serverit
+Comment[fi]=Mahdollistaa työskentelyn OpenChange- tai Exchange-palvelimella
 Comment[fr]=Permet de collaborer avec un serveur OpenChange ou Exchange
 Comment[ga]=Breiseán lenar féidir oibriú le freastalaí OpenChange / Exchange
 Comment[gl]=Permite traballar cun servidor OpenChange / Exchange
--- runtime/resources/ical/icalresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/ical/icalresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=ICal Calendar File
 Name[es]=Archivo de calendario ICal
 Name[et]=ICali kalendrifail
+Name[fi]=iCal-kalenteritiedosto
 Name[fr]=Fichier de calendrier au format ICal
 Name[ga]=Comhad Féilire ICal
 Name[gl]=Ficheiro de calendario de ICal
@@ -48,6 +49,7 @@
 Comment[en_GB]=Loads data from an iCal file
 Comment[es]=Carga datos de un archivo iCal local
 Comment[et]=Andmete laadimine iCal-failist
+Comment[fi]=Noutaa tietoa iCal-tiedostosta
 Comment[fr]=Charge des données depuis un fichier au format iCal
 Comment[ga]=Breiseán a luchtaíonn sonraí ó chomhad iCal
 Comment[gl]=Carrega datos desde un ficheiro iCal
--- runtime/resources/ical/notes/notesresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/ical/notes/notesresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -18,7 +18,7 @@
 Name[et]=Sedelid
 Name[eu]=Oharrak
 Name[fa]=یادداشتها
-Name[fi]=Muistio
+Name[fi]=Muistiinpanot
 Name[fr]=Notes
 Name[fy]=Notysjes
 Name[ga]=Nótaí
@@ -73,6 +73,7 @@
 Comment[en_GB]=Loads data from a notes file
 Comment[es]=Carga datos desde un archivo de notas
 Comment[et]=Andmete laadimine sedelite failist
+Comment[fi]=Noutaa tietoa muistiinpanotiedostosta
 Comment[fr]=Charge des données depuis un fichier notes
 Comment[ga]=Breiseán a luchtaíonn sonraí ó chomhad nótaí
 Comment[gl]=Carrega datos desde un ficheiro de notas
--- runtime/resources/nepomuktag/nepomuktagresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/nepomuktag/nepomuktagresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=Nepomuk Tags (Virtual Folders)
 Name[es]=Etiquetas Nepomuk (Carpetas virtuales)
 Name[et]=Nepomuki sildid (virtuaalkaustad)
+Name[fi]=Nepomuk-tunnisteet (virtuaalikansiot)
 Name[fr]=Étiquettes Nepomuk (dossiers virtuels)
 Name[gl]=Etiquetas Nepomuk (cartafoles virtuais)
 Name[hu]=Nepomuk-címkék (virtuális mappák)
@@ -46,6 +47,7 @@
 Comment[en_GB]=Virtual folders for selecting messages which have Nepomuk Tags.
 Comment[es]=Carpetas virtuales para seleccionar mensajes que tienen etiquetas Nepomuk.
 Comment[et]=Virtuaalkaustad Nepomuki siltidega kirjade valimiseks.
+Comment[fi]=Virtuaalikansiot Nepomuk-tunnisteisten tiedostojen valitsemiseksi.
 Comment[fr]=Des dossiers virtuels qui permettent de sélectionner les messages avec des étiquettes Nepomuk.
 Comment[gl]=Cartafoles virtuais para escoller mensaxes que teñan etiquetas de Nepomuk.
 Comment[hu]=Virtuális mappák adott Nepomuk-címkéket tartalmazó üzenetek megjelenítéséhez.
--- runtime/resources/kcal/kcalresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/kcal/kcalresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -10,6 +10,7 @@
 Name[en_GB]=KDE Calendar (traditional)
 Name[es]=Calendario de KDE (tradicional)
 Name[et]=KDE kalender (traditsiooniline)
+Name[fi]=KDE:n kalenteri (perinteinen)
 Name[fr]=Calendrier KDE (traditionnel)
 Name[ga]=Féilire KDE (traidisiúnta)
 Name[gl]=Calendario do KDE (tradicional)
@@ -48,6 +49,7 @@
 Comment[en_GB]=Loads data from a traditional KDE calendar resource
 Comment[es]=Carga datos desde un recurso tradicional de Calendario de KDE
 Comment[et]=Andmete laadimine traditsioonilisest KDE kalendri ressursist
+Comment[fi]=Noutaa tietoa KDE:n perinteisestä kalenteriresurssista
 Comment[fr]=Charge des données depuis un calendrier KDE traditionnel
 Comment[ga]=Breiseán a luchtaíonn sonraí ó acmhainn thraidisiúnta fhéilire KDE
 Comment[gl]=Carrega datos desde un recurso de calendario tradicional do KDE
--- runtime/resources/microblog/microblog.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/microblog/microblog.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -9,6 +9,7 @@
 Name[en_GB]=Microblog (Twitter and Identi.ca)
 Name[es]=Microblog (Twitter e Identi.ca)
 Name[et]=Mikroblogi (Twitter ja Identi.ca)
+Name[fi]=Mikroblogaus (Twitter ja Identi.ca)
 Name[fr]=Microblog (Twitter et Identi.ca)
 Name[ga]=Micreabhlagadóireacht (Twitter agus Identi.ca)
 Name[gl]=Microblog (Twitter e Identi.ca)
@@ -44,6 +45,7 @@
 Comment[en_GB]=Shows your microblog data from Twitter or Identi.ca.
 Comment[es]=Muestra sus datos microblog desde Twitter o Identi.ca.
 Comment[et]=Näitab mikroblogi andmeid Twitterist või Identi.ca-st
+Comment[fi]=Näyttää mikroblogitietosi Twitteristä tai Identi.casta.
 Comment[fr]=Affiche vos informations de Twitter ou Identi.ca
 Comment[gl]=Mostra os seus datos de microblog de Twitter ou Identi.ca.
 Comment[hu]=Megjeleníti a Twitter vagy az Identi.ca adatait.
--- runtime/resources/contacts/contactsresource.desktop	(.../tags/KDE/4.4.0/kdepim)	(wersja 1091895)
+++ runtime/resources/contacts/contactsresource.desktop	(.../branches/KDE/4.4/kdepim)	(wersja 1091895)
@@ -8,6 +8,7 @@
 Name[en_GB]=Personal Contacts
 Name[es]=Contactos personales
 Name[et]=Isiklikud kontaktid
+Name[fi]=Henkilökohtaiset yhteystiedot
 Name[fr]=Contacts personnels
 Name[gl]=Contactos persoais
 Name[it]=Contatti personali
@@ -37,6 +38,7 @@
 Comment[en_GB]=The address book with personal contacts
 Comment[es]=La libreta de direcciones con contactos personales
 Comment[et]=Isiklikke kontakte sisaldav aadressiraamat
+Comment[fi]=Henkilökohtaisten yhteystietojen osoitekirja
 Comment[fr]=Le carnet d'adresses avec vos contacts personnels
 Comment[gl]=O caderno de enderezos cos contactos persoais
 Comment[it]=La rubrica con i contatti personali
